java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -503,6 +503,10 @@ protected String handleStringResponse(Response response) throws RedisException {
         return handleRedisResponse(String.class, EnumSet.of(ResponseFlags.ENCODING_UTF8), response);
     }
 
+    protected GlideString handleStringResponseBinary(Response response) throws RedisException {
```

Comment:
use handleGlideStringResponse and delete this

Reply:
Will do in new commit